Update to work with Cesium 1.59

Flowm opened the pull request:
The plugin fails in Cesium 1.59 with `RuntimeError: Fragment shader failed to compile.  Compile log: ERROR: 0:2: 'czm_ellipsoid' : syntax error`

`czm_ellipsoid` and thus `czm_ellipsoid.inverseRadii` were replaced by constants in https://github.com/AnalyticalGraphicsInc/cesium/pull/7944.
Use the new `czm_ellipsoidInverseRadii` instead to make the plugin work with Cesium 1.59.

jlouns committed:
Updating sensor visualizers to work with entity.show/entity.isShowing properties in Cesium 1.8

jlouns committed:
Updating plugin to work with Cesium 1.15

jlouns committed:
Removing third party libs in preparation for bower

jlouns committed:
Moving to bower for dependency management

jlouns committed:
Updating README

jlouns committed:
Upgrading build process to gulp

jlouns committed:
Renaming directories

jlouns committed:
Fixing use of minified shader files in build

jlouns committed:
Fixing inclusion of license comments in minified dist

jlouns committed:
Changing test file naming convention

jlouns committed:
Changing script naming convention

jlouns committed:
Creating module structure

jlouns committed:
Adding ability to skip webgl tests for CI builds

jlouns committed:
Adding travis-ci integration

jlouns committed:
Adding custom chrome config for travis build

jlouns committed:
Fixing camelcase issue

jlouns committed:
Moving to electron for ci builds

jlouns committed:
Adding npmignore

jlouns committed:
Renaming to cesium-sensor-volumes to avoid confusion with original plugin

jlouns committed:
Updating readme and links

jlouns committed:
Adding keywords

jlouns committed:
1.16.1

jlouns committed:
Adding version and dependency badges

jlouns committed:
Updating to cesium 1.17

jlouns committed:
1.17.0

jlouns committed:
Upgrading to cesium 1.18

jlouns committed:
1.18.0

jlouns committed:
Upgrading to cesium 1.19

jlouns committed:
1.19.0

jlouns committed:
Upgrading to cesium 1.20 and requirejs 2.2.0

jlouns committed:
1.20.0

jlouns committed:
Upgrading to cesium 1.21

jlouns committed:
1.21.0

jlouns committed:
Upgrading to cesium 1.22

jlouns committed:
1.22.0

jlouns committed:
Upgrading to cesium 1.23

jlouns committed:
1.23.0

jlouns committed:
Upgrading to cesium 1.24

jlouns committed:
1.24.0

jlouns committed:
Upgrading to cesium 1.25

jlouns committed:
1.25.0

jlouns committed:
Upgrading to cesium 1.26

jlouns committed:
1.26.0

jlouns committed:
Upgrading to cesium 1.27

jlouns committed:
1.27.0

jlouns committed:
Upgrading to cesium 1.28

jlouns committed:
1.28.0

jlouns committed:
Identifying initialize test as requiring webgl to fix CI build

jlouns committed:
Updating dependencies

jlouns committed:
Upgrading to cesium 1.29

jlouns committed:
1.29.0

jlouns committed:
Upgrading to cesium 1.30

jlouns committed:
1.30.0

jlouns committed:
updating to cesium 1.31

jlouns committed:
1.31.0

jlouns committed:
Updating dependencies

jlouns committed:
Updating requirejs

jlouns committed:
Upgrading to cesium 1.32.1

jlouns committed:
1.32.0

jlouns committed:
Updating node version in CI config

Flowm committed:
Update to work with Cesium 1.59

icemagno commented:
I face an already known bug in Chrome here. The original bug is https://github.com/AnalyticalGraphicsInc/cesium/issues/7798

Flowm commented:
@icemagno Bug seems to be resolved in latest Chrome version.
@xagriff Could you have a look and merge?

icemagno commented:
Hi xagriff. Since I can't see any practical use for the sensors I have no using it anymore. I think I could make some kind of calculations like viewshed or detections but as far I can see it is just a dead object.

damiendt committed:
Convert file eol from crlf to lf (unix style)

damiendt committed:
Updated to support Cesium 1.68
Changed defineProperties to Object.defineProperties.
Changed czm_phong to new function signature with 3 parameters.

Flowm committed:
Upgrade to gulp 4

Flowm committed:
Add ConicSensor example

Flowm committed:
Update to Cesium 1.77

Flowm committed:
Merge pull request #1 from damiendt/cesium-sensor-volumes

Cesium sensor volumes

Flowm committed:
Fix lint warnings

Flowm committed:
Transform codebase to es6

Flowm committed:
Refactor build system and add es6 module build

Flowm committed:
Update readme

Flowm committed:
Add dist files

Flowm committed:
Upgrade to Cesium 1.99

Flowm committed:
Upgrade dev dependencies

Flowm committed:
Upgrade to Cesium 1.100

Flowm committed:
Port shaders to WebGL2

Flowm committed:
Upgrade to Cesium 1.102

Flowm committed:
Upgrade to Cesium 1.133 and modernize Cesium imports to use ES6 destructured syntax

Flowm committed:
Upgrade to Cesium 1.134 and replace deprecated defaultValue function

Flowm commented:
Closing after it was open for 6+ years
